wire-schema/src/test/java/com/squareup/wire/internal/protoparser/ServiceElementTest.java

```diff
@@ -30,55 +45,55 @@
       ServiceElement.builder().name(null);
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("name == null");
+      assertThat(e).hasMessage("name");
     }
     try {
       ServiceElement.builder().qualifiedName(null);
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("qualifiedName == null");
+      assertThat(e).hasMessage("qualifiedName");
     }
     try {
       ServiceElement.builder().documentation(null);
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("documentation == null");
+      assertThat(e).hasMessage("documentation");
     }
     try {
       ServiceElement.builder().addRpc(null);
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("rpc == null");
+      assertThat(e).hasMessage("rpc");
     }
     try {
       ServiceElement.builder().addRpcs(null);
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("rpcs == null");
+      assertThat(e).hasMessage("rpcs");
     }
     try {
       ServiceElement.builder().addRpcs(Collections.<RpcElement>singleton(null));
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("rpc == null");
+      assertThat(e).hasMessage("rpc");
     }
     try {
       ServiceElement.builder().addOption(null);
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("option == null");
+      assertThat(e).hasMessage("option");
     }
     try {
       ServiceElement.builder().addOptions(null);
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("options == null");
+      assertThat(e).hasMessage("options");
     }
     try {
       ServiceElement.builder().addOptions(Collections.<OptionElement>singleton(null));
       fail();
     } catch (NullPointerException e) {
-      assertThat(e).hasMessage("option == null");
+      assertThat(e).hasMessage("option");
```

Comment:
Guava's `checkNotNull()` wants it to be just the field name. I have a preference for the expression, but not enough to avoid Guava.